lib/kube/grpc/utils.go

```diff
@@ -42,15 +43,23 @@ import (
 // multiplexing mode, the Kube proxy is always reachable on the same address as
 // the web server using the SNI.
 func getWebAddrAndKubeSNI(proxyAddr string) (string, string, error) {
-	addr, port, err := utils.SplitHostPort(proxyAddr)
+	addr, port, err := net.SplitHostPort(proxyAddr)
```

Comment:
I know we have test coverage, but just so someone doesn't come along and waste time trying to consolidate.

```suggestion
	// we avoid using utils.SplitHostPort because
	// we allow the host to be empty
	addr, port, err := net.SplitHostPort(proxyAddr)
```